lib/chef-dk/command/verify.rb

```diff
@@ -141,6 +147,35 @@ def components
           # ------------
           failures = []
 
+          # fail the verify if we have more than one version of chef-provisioning or any of its drivers.
```

Comment:
@danielsdeleo Yeah... that seems like enough work (comparing against the manifest, I'm guessing) that just leaving the existing code and turning it into a warning looks like a better idea for the moment. Does that sound like a good path into 0.8.0?